feature_details.md

```diff
@@ -0,0 +1,174 @@
+
+### Multi Page Apps (MPA)
+It’s not possible to have these transitions today for multi-page apps (or same-origin navigations) and in fact a lack of this ends up being one of the drivers for web authors to create SPAs. We want this API to enable authoring transitions within an MPA with a similar feature set as the SPA case above. 
+
+### Cross Origin Transitions
+Cross-origin transitions are infeasible similar to MPAs. A common use-case for this is content aggregator sites (search engines, social media sites, news aggregators, etc.). These sites frequently display a hero image/header which could be animated during the navigation to provide a seamless transition.
+
+Cross-origin transitions have additional security/privacy constraints. In particular we must ensure the feature does not allow any cross-site information sharing which imposes the following restrictions :
+
+* The animations associated with the transition need to be defined completely by the previous Document, which is aware of the site the user is navigating to and has contextual information about the user journey.
+
+* The animations need to be executed such that there are no observable side-effects for the new and previous Document. For example, setting an obscure animation for a specific element on the new Document could be used to exchange information between origins. Enforcing this could influence the stage in the rendering pipeline where the UA executes these animations.
+
+An important assumption in the design of this feature is that it's reasonable to limit customization options for cross-origin transitions to make it easier for UAs to enforce the constraints above. But the API/implementation should strive to not unnecessarily limit capabilities for same-origin transitions due to cross-origin constraints.
+
+
+## Design
+**Disclaimer** : The details in this design focus on the SPA/MPA use-case with extensive customizability. It’s expected that the functionality will be delivered incrementally with the initial version of the API being quite limited in scope. The purpose of exploring the customizability options is to ensure the API can evolve to accommodate them.
+
+There are 2 kinds of semantic transitions which are a part of this feature :
+
+* Enter/Exit Transition : An element which exists only on the new Document animates as it enters the screen. The final state of the element in the new Document provides an end state for the animatable properties. Similarly an element which exists only on the previous Document animates as it exits the screen, with a start state for the animatable properties.
```

Comment:
Just a general comment, we could probably use a glossary document / section so that we can agree whether it's called previous/new Document or incoming/outgoing Document or something else (and same for other terms that may come up)

Reply:
Good point. Let's stick with incoming/outgoing for now and I'll clarify this in a Glossary Section.